resources/lang/de/Config.properties

```diff
@@ -164,6 +164,12 @@ itemLoadingAnimationTypeBars = Ladeanimation mit Balken
 itemLoadingAnimationTypeSpinner = Ladeanimation mit Spinner
 itemShowCategoryImageLabel = Vorschaubild in der Kategorieansicht anzeigen
 itemShowCategoryDescriptionLabel = Beschreibung der Kategorie in der Kategorieansicht anzeigen
+itemShowCategoryDescriptionTopLabel = Kategoriebeschreibung über Artikelliste anzeigen
+itemShowCategoryDescriptionBottomLabel = Kategoriebeschreibung unter Artikelliste anzeigen
+itemShowCategoryDescriptionNone = Keine
+itemShowCategoryDescription1 = Beschreibung 1
+itemShowCategoryDescription2 = Beschreibung 2
+itemShowCategoryDescriptionBoth = Beide
```

Comment:
Wir haben bisher Übersetzungen mit "both" immer mit einem "und" übersetzt:
headerShowNavbarsBoth = Oben und links

Reply:
Also vielleicht "Beschreibung 1 und 2" etwas in der Art

Reply:
wird angepasst